tests/account.test.js

```diff
@@ -0,0 +1,46 @@
+// account.test.js
+import { describe, it, expect } from 'vitest';
+import { Account } from '../src/model/Account.js';
+
+describe('Account', () => {
+  it('Criacao de uma conta valida', () => {
+    const account = new Account('Artur Seppa', 100);
+    
+    expect(account).toBeDefined();
+    expect(account.getName()).toBe('Artur Seppa');
+    expect(account.getBalance()).toBe(100);
+    expect(account.getNumAccount()).toBe('000001');
+    expect(account.getCreatedAt()).toBeInstanceOf(Date);
+  });
+
+  it('Geracao de um numero de conta no formato correto', () => {
+    const account = new Account('Artur Seppa');
+    
+    expect(account.getNumAccount()).toMatch(/^\d{6}$/);
+  });
+
+  it('Deve criar uma conta com valor default igual a 0', () => {
+    const account = new Account('Artur Seppa');
+    
+    expect(account.getBalance()).toBe(0);
+  });
+
+  it('Erro para name vazio', () => {
+    expect(() => new Account('')).toThrow('O nome não pode ser vazio.');
+    expect(() => new Account('  ')).toThrow('O nome não pode ser vazio.');
+    expect(() => new Account()).toThrow('O nome não pode ser vazio.');
+  });

```

Comment:
nice testes! but there are still some edge cases that could be tested, such as attempting to update an account with an empty name